src/mds/Server.cc

```diff
@@ -2636,6 +2636,16 @@ void Server::dispatch_client_request(MDRequestRef& mdr)
     dout(1) << " unknown client op " << req->get_op() << dendl;
     respond_to_request(mdr, -CEPHFS_EOPNOTSUPP);
   }
+
+  // HACK: Always release the capability here so that the MDS is no longer tracking it.
+  client_t client = mdr->get_client();
+  CInode* ino = mdr->in[0];
+  if (ino) {
+    Capability *cap = ino->get_client_cap(client);
+    if (cap) {
+      mds->locker->_do_cap_release(client, inodeno_t((uint64_t)cap->get_inode()) , cap->get_cap_id(), cap->get_mseq(), cap->get_last_seq());
+    }
+  }
```

Comment:
Looks like my MDS is starting and things seem to be functioning; but I'm not sure yet this is doing what I intend.
Is there an administrative call to dump out all capabilities for every client known to the MDS ?
(Googling brings up the CephFS Auth which is not the same thing)

Reply:
Keep in mind the client may also get capabilities via `readdir`. I think this hack would work better in Locker where caps are issued.